spec/models/resource_spec.rb

```diff
@@ -18,4 +18,39 @@
   end
 
   it_behaves_like 'indexable', :resource
+
+  describe 'privacy' do
+    let(:resource) { create(:resource, privacy: :priv) }
+
+    describe 'after_commit on: [:create]' do
+      it "adds the #{name} to the search index after creation" do
+        allow(AddToIndexJob).to receive(:perform_async)
```

Comment:
Yup, else I'm getting:

```
#<AddToIndexJob (class)> expected to have received perform_async, but that object is not a spy or method has not been stubbed.
```

We need to setup the stubbing before the expectation.